apps/web-staking/src/app/components/pool/NewPoolComponent.tsx

```diff
@@ -1,6 +1,7 @@
 import PoolTextComponent from "./PoolTextComponent";
 import { ExternalLinkComponent } from "@/app/components/ui/links/ExternalLink";
-import { ConnectButton, PrimaryButton } from "@/app/components/ui/buttons";
+import { ConnectButton } from "@/app/components/ui/buttons";
+import PoolDropdownComponent from "./PoolDropdownComponent";
 
 interface NewPoolProps {
```

Comment:
What are new pool props ?

Reply:
This component receives methods onOpen, which allows to connect to the wallet, wallet address and isApproved status, which confirms KYC or not

Reply:
Please rename this accordingly.

Reply:
Done